Tutorials/Day_3/Tuto_3.3_Discovering_and_using_published_posterior_samples.ipynb

```diff
@@ -156,6 +156,7 @@
     "# if you do not have wget installed, simply download manually \n",
     "# https://dcc.ligo.org/LIGO-P1800370/public/GW150914_GWTC-1.hdf5 \n",
     "# from your browser\n",
+    "# Learn more about this dataset here: https://dcc.ligo.org/LIGO-P1800370/public \n",
```

Comment:
Is there a reason for the space at the end of the line ? Is it because of the URL checker ?

Reply:
yeap, I had to add a space or it would trip the link checker, which is unfortunate but I don't know a better solution to the problem.

Reply:
If I'm correct, without space, lychee would try `https://dcc.ligo.org/LIGO-P1800370/public/n`, right ? We have some examples like that in `.lycheeignore` so adding one more should do the trick and be less fragile.

Reply:
I understand your point, but I have to let you notice that if we add every link to the ignore list we're defeating the purpose having a link checker, which is to prevent link rot. And this will happen with every link we add to any notebook. There's also the other point that we don't modify the content often enough, IMHO checking that links don't rot beats purity of the notebooks. In any case, it's your call.

Reply:
You're right.😅 However, such cases are supposed to be a bug (lycheeverse/lychee#1659) so it should work at some point (although the issue is not very active). So I think we should remove the space and add this as an exception.

Reply:
Now I think that maybe I misunderstood, did you say I should keep the space in the notebook but add `https://dcc.ligo.org/LIGO-P1800370/public/n` to `lycheeignor`? Or do you want me to keep the space but add the link nevertheless? If you meant the former I'll send a commit soon.

Reply:
I meant: remove the space but add `https://dcc.ligo.org/LIGO-P1800370/public/n` to `.lycheeignore`. Is it clearer ?

Reply:
I see that `https://iopscience.iop.org/article/10.1088/0264-9381/33/21/215004/pdf` returns 403. I have the same problem in another PR (#55). I wonder if this is new.

Reply:
I think it's new. Feels like a whack-a-mole game.